openmp/libomptarget/plugins-nextgen/host/CMakeLists.txt

```diff
@@ -14,36 +14,36 @@ if(CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64le$")
 endif()
 
 # Create the library and add the default arguments.
-add_target_library(omptarget.rtl.${machine} ${machine})
+add_target_library(omptarget.rtl.host ${machine})
```

Comment:
can you just set a variable and then use it everywhere else?

Reply:
The main reason for this change is in https://github.com/llvm/llvm-project/pull/87009 where I wanted to be able to enable these individually with a normal text string. E.g. `-DLIBOMPTARGET_PLUGINS_TO_BUILD=cuda;amdgpu;host`. This doesn't work if the name changes on the target since I wanted to make everything obey a standard naming convention.

Reply:
I mean something like this:
```
set (host_target_name omptarget.rtl.host)
...
target_link_libraries(${host_target_name} PRIVATE FFI::ffi_static)
```

Reply:
Ah, well I figured it's all the same since it's the same name throughout the file. Not much difference between `host_target_name` and `omptarget.rtl.host` being put on every cmake argument.

Reply:
If we want to rename it later, having in a variable would be easier though.